documentation/modules/exploit/multi/http/lesplunk_upload_app_exec.md

```diff
@@ -0,0 +1,83 @@
+## Vulnerable Application
```

Comment:
What is le splunk and how does it differ from splunk?

Reply:
I am not sure I fully understood your comment: which comparison are you intending here?

Reply:
What's the difference between these two files?

* documentation/modules/exploit/multi/http/lesplunk_upload_app_exec.md
* documentation/modules/exploit/multi/http/splunk_upload_app_exec.md

Reply:
the former is a duplicate. fixed now :) thanks